src/components/listItem/ListItem.styles.js

```diff
@@ -0,0 +1,37 @@
+import {StyleSheet} from 'react-native'
+import { size, color } from "../../theme";
+
+export const styles = StyleSheet.create({
+    listItemContainer:{
+        backgroundColor:color.white,
+        borderRadius:size.sm,
+        flexDirection:'row',
+        justifyContent:'space-between',
+        paddingHorizontal:size.rg,
+        paddingVertical:size.sm,
+        alignItems:'center',
+    },
+    infoContainer:{
+        width:'70%',
```

Comment:
You can set to info container flex:1


And it will push icon, and takes rest of the screen. Won't be need to set 30% for icon. It will have just icon fixed size